README.md

```diff
@@ -34,15 +34,10 @@ metadata:
     app.kuberentes.io/managed-by: kustomize
     app.kubernetes.io/created-by: application-connector-manager
   name: applicationconnector-sample
-spec:
-  disableLegacyConnectivity : "false"
```

Comment:
Example of fully configured CR: 

```
domainName: c-9057319.dev.kyma.ondemand.com
appConnValidator:
  logLevel: "info"
  logFormat: "json"
appGateway:
  proxyTimeout: "10s"
  requestTimeout: "10s"
  logLevel: "info"
```

All parametes are optional. The simplest spec is 

`spec: {}`

Reply:
For now, I added the simplest spec `spec: {}`